indico/modules/events/papers/templates/management/cfp_actions/ended.html

```diff
@@ -0,0 +1,28 @@
+{% extends 'events/papers/management/cfp_actions/_base.html' %}
+
+{% block label %}
+    {% trans -%}
+        The call for papers is closed
```

Comment:
Not a strong opinion either, but this could easily fit in one line and you avoid using the `-`:
```
{% trans %}The call for papers is closed{% endtrans %}
```